### PR TITLE
Runbook for cloudwatch networking alarms

### PR DESCRIPTION
**Pull request by dms1981**
This PR adds a runbook covering our networking-related CloudWatch alarms, with some discussion on what can cause the alerts to trigger, and what actions can be taken in the event that an alert has triggered.

### CODE REVIEW

**Comment by dms1981** [comment]
This PR resolves #2635

File: source/runbooks/cloudwatch-networking-alarms.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+This alarm triggers when no traffic has been seen traversing a VPC Transit Gateway attachment and exists on a per-VPC
+basis. The intent for this alarm was to alert us when a change or event occurred that caused a cessation of traffic from
+a VPC to the Modernisation Platform Transit Gateway.
+
+Should this alarm trigger you should confirm the VPC in question and assess the behaviour of traffic over the alarm
+period, extending past this alarm to observe what normal behaviour should look like. It is possible for this alarm to
+trigger during ordinary operation when the polling window is short; no traffic over five minutes is not uncommon.
+
+In the event that this alarm triggers, and it has been deemed legitimate, your next step should be to confirm that there
```
Comment (julialawrence) [inline source/runbooks/cloudwatch-networking-alarms.html.md.erb:61]: What's the best way to know if it's legitimate?

Comment (dms1981) [reply source/runbooks/cloudwatch-networking-alarms.html.md.erb:61]: It's something to be judged on a case by case basis, but I'll put an example of a legitimate trigger and a false positive in if that helps?

Comment (julialawrence) [reply source/runbooks/cloudwatch-networking-alarms.html.md.erb:61]: It would really help thank you!

**Review by julialawrence — APPROVED** [review]
Thank you! This will make it easier to know how to handle alarms!

**Pull request merged**